drivers/sqljs-driver/src/main/kotlin/app/cash/sqldelight/driver/sqljs/JsSqlDriver.kt

```diff
@@ -136,4 +143,12 @@ private class JsSqlPreparedStatement : SqlPreparedStatement {
   override fun bindString(index: Int, string: String?) {
     parameters.add(string)
   }
+
+  override fun bindBoolean(index: Int, boolean: Boolean?) {
+    parameters.add(when (boolean) {
+      null -> null
+      true -> 1.toDouble()
+      false -> 0.toDouble()
```

Comment:
```suggestion
      true -> 1.0
      false -> 0.0
```
I'm not sure the original is a compile-time conversion